packages/nextjs/services/store/tokenStore.ts

```diff
@@ -475,7 +483,7 @@ export const fetchTokenPairsData = async () => {
   const chain = await getChainId();
 
   try {
-    await loadPredefinedValues("https://redact-resources.s3.eu-west-1.amazonaws.com/predefined-token-list.json");
+    await loadPredefinedValues("http://redact-resources.s3.eu-west-1.amazonaws.com/predefined-token-list.json");
```

Comment:
There was a caching issue with the S3 bucket, this is a temporary fix which can be removed at some point. Is there somewhere I can force a cache miss in S3 im not super familiar